### PR TITLE
Add food props batch3

### PR DESCRIPTION
**Pull request by sumitnegi7**
@qur786  this PR can be reviewed by the resource given in #4

### CODE REVIEW

File: src/data/food-compounds.ts [hunk 1]
```diff
@@ -3078,4 +3078,1077 @@ export const FoodCompounds: Record<string, FoodProps> = {
...
+    vaporPressureConstantEqnNumber: 0,
+    vaporPressureConstantA: 0,
+    vaporPressureConstantB: 0,
+    vaporPressureConstantC: 0,
+    vaporPressureConstantD: 0,
+    vaporPressureConstantE: 0,
+    idealGasHeatCapacityEqnNumber: 4,
+    idealGasHeatCapacityConstantA: 28161,
+    idealGasHeatCapacityConstantB: 435.7,
+    idealGasHeatCapacityConstantC: -0.25,
```
Comment (maneetgoyal) [inline src/data/food-compounds.ts:3111]:
```suggestion
    idealGasHeatCapacityConstantC: -0.2508,
```

File: src/data/food-compounds.ts [hunk 2]
```diff
@@ -3078,4 +3078,1077 @@ export const FoodCompounds: Record<string, FoodProps> = {
...
+    HfusAtTfkJmol: 37.7,
+    densityTC: 0,
+    densitygmL: 0,
+    elements: [
+      { name: 'C', text: '14' },
+      { name: 'H', text: '28' },
+      { name: 'O', text: '2' },
+    ],
+  },
+
```
Comment (maneetgoyal) [inline src/data/food-compounds.ts:4020]: Palmitic Acid is missed.

File: src/data/food-compounds.ts [hunk 3]
```diff
@@ -3078,4 +3078,1077 @@ export const FoodCompounds: Record<string, FoodProps> = {
...
+    liquidViscocityConstantC: 0,
+    liquidViscocityConstantD: 0,
+    liquidViscocityConstantE: 0,
+    liquidDensityEqnNumber: 0,
+    liquidDensityConstantA: 0,
+    liquidDensityConstantB: 0,
+    liquidDensityConstantC: 0,
+    liquidDensityConstantD: 0,
+    liquidDensityConstantE: 0,
+    vaporViscocityEqnNumber: 0,
```
Comment (qur786) [inline src/data/food-compounds.ts:3947]: "Vapor_Viscosity_EqNo": "\n    "

File: src/data/food-compounds.ts [hunk 4]
```diff
@@ -3078,4 +3078,1077 @@ export const FoodCompounds: Record<string, FoodProps> = {
...
+    vaporViscocityConstantA: 0,
+    vaporViscocityConstantB: 0,
+    vaporViscocityConstantC: 0,
+    vaporViscocityConstantD: 0,
+    vaporViscocityConstantE: 0,
+    vaporThermalConductivityEqnNumber: 0,
```
Comment (qur786) [inline src/data/food-compounds.ts:3953]: "Vapor_Thermal_Conductivity_EqNo": "\n    "

File: src/data/food-compounds.ts [hunk 5]
```diff
@@ -3078,4 +3078,1077 @@ export const FoodCompounds: Record<string, FoodProps> = {
...
+    blackOilPNAA: 0,
+    blackOilPNAN: 0,
+    blackOilPNAP: 0,
+    ion: false,
+    salt: false,
+    hydratedSalt: false,
+    hydrationNumber: 0,
+    charge: 0,
+    positiveIon: 0,
+    negativeIon: 0,
```
Comment (qur786) [inline src/data/food-compounds.ts:4003]: "PositiveIon": "\n    ",
        "NegativeIon": "\n    ",

File: src/data/food-compounds.ts [hunk 6]
```diff
@@ -3078,4 +3078,1077 @@ export const FoodCompounds: Record<string, FoodProps> = {
...
+    liquidViscocityConstantC: 0,
+    liquidViscocityConstantD: 0,
+    liquidViscocityConstantE: 0,
+    liquidDensityEqnNumber: 0,
+    liquidDensityConstantA: 0,
+    liquidDensityConstantB: 0,
+    liquidDensityConstantC: 0,
+    liquidDensityConstantD: 0,
+    liquidDensityConstantE: 0,
+    vaporViscocityEqnNumber: 0,
```
Comment (qur786) [inline src/data/food-compounds.ts:4081]: "Vapor_Viscosity_EqNo": "\n    "

File: src/data/food-compounds.ts [hunk 7]
```diff
@@ -3078,4 +3078,1077 @@ export const FoodCompounds: Record<string, FoodProps> = {
...
+    vaporViscocityConstantA: 0,
+    vaporViscocityConstantB: 0,
+    vaporViscocityConstantC: 0,
+    vaporViscocityConstantD: 0,
+    vaporViscocityConstantE: 0,
+    vaporThermalConductivityEqnNumber: 0,
```
Comment (qur786) [inline src/data/food-compounds.ts:4087]: "Vapor_Thermal_Conductivity_EqNo": "\n    "

File: src/data/food-compounds.ts [hunk 8]
```diff
@@ -3078,4 +3078,1077 @@ export const FoodCompounds: Record<string, FoodProps> = {
...
+    blackOilPNAA: 0,
+    blackOilPNAN: 0,
+    blackOilPNAP: 0,
+    ion: false,
+    salt: false,
+    hydratedSalt: false,
+    hydrationNumber: 0,
+    charge: 0,
+    positiveIon: 0,
+    negativeIon: 0,
```
Comment (qur786) [inline src/data/food-compounds.ts:4137]: "PositiveIon": "\n    ",
        "NegativeIon": "\n    ",

File: src/data/food-compounds.ts (hunk 2 above)
Comment (sumitnegi7) [reply src/data/food-compounds.ts:4020]: Sir it is already added in batch1 we dicussed this over call yesterday.

File: src/data/food-compounds.ts [hunk 9]
```diff
@@ -3078,4 +3078,1077 @@ export const FoodCompounds: Record<string, FoodProps> = {
...
+    liquidViscocityConstantC: 0,
+    liquidViscocityConstantD: 0,
+    liquidViscocityConstantE: 0,
+    liquidDensityEqnNumber: 0,
+    liquidDensityConstantA: 0,
+    liquidDensityConstantB: 0,
+    liquidDensityConstantC: 0,
+    liquidDensityConstantD: 0,
+    liquidDensityConstantE: 0,
+    vaporViscocityEqnNumber: 0,
```
Comment (qur786) [inline src/data/food-compounds.ts:3142]: In #4 vapor_Viscosity_eqNo is given as "\n ". I don't what should be there , so I am pointing out it

**Review by qur786 — CHANGES_REQUESTED** [review]
for those values where data was given as "\n " in #4 comment , I know 0 would most probably be the value but since  I don't know what should be there so  I just pointed it out.

File: src/data/food-compounds.ts [hunk 10]
```diff
@@ -3078,4 +3078,1077 @@ export const FoodCompounds: Record<string, FoodProps> = {
...
+    vaporViscocityConstantA: 0,
+    vaporViscocityConstantB: 0,
+    vaporViscocityConstantC: 0,
+    vaporViscocityConstantD: 0,
+    vaporViscocityConstantE: 0,
```
[the remark this answers is not in the document]
Comment (sumitnegi7) [reply src/data/food-compounds.ts:3147]: @qur786  If it is not a part of the interface you can ignore that and in every instance it is stored as `\n` only so it is not useful data.

File: src/data/food-compounds.ts (hunk 2 above)
Comment (maneetgoyal) [reply src/data/food-compounds.ts:4020]: Cool

**Review by maneetgoyal — APPROVED** [review]

**Review by qur786 — APPROVED** [review]

**Pull request merged**